static/app/views/organizationIntegrations/sentryAppExternalForm.tsx

```diff
@@ -252,10 +253,16 @@ export class SentryAppExternalForm extends Component<Props, State> {
 
     if (fieldToPass.type === 'select') {
       // find the options from state to pass down
-      const defaultOptions = (field.choices || []).map(([value, label]) => ({
+      let defaultOptions = (field.choices || []).map(([value, label]) => ({
         value,
         label,
       }));
+      // there are no choices if skip_load_on_open is set, just use what's in the defaultResetValue
+      if (field.skip_load_on_open === true) {
```

Comment:
I'd do 
```tsx
const defaultOptions = field.skip_load_on_open
        ? [{value: defaultResetValues[field.name], label: defaultResetValues[field.name]}]
        : (field.choices || []).map(([value, label]) => ({value, label}));
```
So that you don't have to iterate over choices when `skip_load_on_open` is true.